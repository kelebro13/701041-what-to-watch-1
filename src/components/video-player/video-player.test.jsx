import renderer from "react-test-renderer";
import VideoPlayer from './video-player';

it(`renders properly`, () => {
  const poster = `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`;
  const sourceMp4 = `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`;

  const tree = renderer
    .create(<VideoPlayer name={``} poster={poster} videoLink={sourceMp4}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
