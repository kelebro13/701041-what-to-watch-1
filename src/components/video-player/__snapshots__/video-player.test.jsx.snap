// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly MovieList component 1`] = `
<video
  autoPlay={true}
  muted={true}
  poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
>
  <source
    src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
    type="video/webm; codecs=\\"vp8, vorbis\\""
  />
  <source
    src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
    type="video/mp4; codecs=\\"avc1.42E01E, mp4a.40.2\\""
  />
</video>
`;
